code-challenges.test.js

```diff
@@ -14,42 +14,181 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
+//const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
-const fibLength2 = 10
+//const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// describe('newFib', () => {
+//   it('takes a number of 6 and 10 and returns an array with the Fibonacci Sequence', () => {
+//     const fibLength1 = 6
+//     // Expected output: [1, 1, 2, 3, 5, 8]
+//     const fibLength2 = 10
+//     // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+//     expect(newFib(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+//   expect(newFib(fibLength1)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+// })
+// })
+
+//RED - Good failure
+// FAIL  ./code-challenges.test.js
+// newFib
+// ✕ takes a number of 6 and 10 and returns an array with the Fibonacci Sequence (1 ms)
+
+// ● newFib › takes a number of 6 and 10 and returns an array with the Fibonacci Sequence
+
+// ReferenceError: newFib is not defined
+
+//   27 |     const fibLength2 = 10
+//   28 |     // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// > 29 |     expect(newFib(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+//      |     ^
+//   30 |   expect(newFib(fibLength1)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+//   31 | })
+//   32 | })
+
+//   at Object.expect (code-challenges.test.js:29:5)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+// Snapshots:   0 total
+// Time:        0.157 s
+// Ran all test suites.
+// error Command failed with exit code 1.
+
+
+
+
 // b) Create the function that makes the test pass.
+//PseudoCode
+//input: a number (6)
+//Output: an array within the fibonacci sequence
+//process:Create a function newFibo, add the parameter of an array. 
+//Create conditional statement that ensures number starts with 2
+//create a variable that will store location of new array
+//iterate through the array using for loop
+//take the [i] and create an equation to add the result of the iteration ?
+//create else statement to return "try again"
```

Comment:
There is a lot of stuff in here that isn't part of your code at all. I would suggest revisiting past material on Javascript fundamentals!